package.json

```diff
@@ -26,6 +26,7 @@
   },
   "devDependencies": {
     "c8": "^7.11.0",
+    "envelope-spec": "github:ssbc/envelope-spec#group-test-vectors",
```

Comment:
https://github.com/ssbc/envelope-spec/pull/23

Reply:
:fire: these vectors should be moved to `private-group-spec`, see PR